### PR TITLE
qemu 3.0.0

### PR DESCRIPTION
**Pull request by evshiron**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/qemu.rb [hunk 1]
```diff
@@ -22,10 +22,12 @@ class Qemu < Formula
   end
 
   devel do
-    url "https://download.qemu.org/qemu-3.0.0-rc4.tar.xz"
-    sha256 "420409d0c47169c4323b29d8af0a14d8276f4f038a74f37891829650dfb9d8cd"
+    url "https://download.qemu.org/qemu-3.0.0.tar.xz"
+    sha256 "8d7af64fe8bd5ea5c3bdf17131a8b858491bcce1ee3839425a6d91fb821b5713"
   end
 
+  option "with-hvf", "Enable Hypervisor.framework support (3.x)"
```
Comment (DomT4) [inline Formula/qemu.rb]: It this is compatible with all the other options we offer in 3.x just turn it on by default, unless there's another good reason not to.

**Review by DomT4 — CHANGES_REQUESTED** [review]
What @SMillerDev said as well, please.

**Comment by evshiron** [comment]
The stable version no longer needs a patch to build when `--with-gtk+3` after upgrading to `3.0.0`, and the `Hypervisor.framework support` is automatically enabled if available when `./configure`, so I remove the patch and the `with-hvf` option.

**Comment by DomT4** [comment]
You can remove the `devel do` block entirely now, because it's older than `stable`.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-core/commit/c7de8968b0b7e872f339812d88942181aee515ec. Thank you for your contribution to Homebrew @evshiron; we appreciate it! :smiley_cat:

**Pull request closed without merging**